src/engine/strat_engine/pool.rs

```diff
@@ -9,14 +9,16 @@ use std::path::Path;
 use std::path::PathBuf;
 use std::vec::Vec;
 
+use rand::random;
+use time::now;
+use uuid::Uuid;
+use serde_json;
+
 use devicemapper::consts::SECTOR_SIZE;
 use devicemapper::DM;
 use devicemapper::{DataBlocks, Sectors, Segment};
 use devicemapper::LinearDev;
-use devicemapper::{ThinPoolDev, ThinPoolStatus, ThinPoolWorkingStatus};
-use time::now;
-use uuid::Uuid;
-use serde_json;
+use devicemapper::{ThinDevId, ThinPoolDev, ThinPoolStatus, ThinPoolWorkingStatus};
```

Comment:
See filesystem.rs. devicemapper was obviously special until this Pr, in which the use of rand was removed, rendering its position ambiguous.